CorinneTest/Person.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+namespace CorinneTest
+{
+    public class Person 
+    {
+        private string name;
+        public string Name
+        {
+            get => this.name; 
+            set { this.name = value; }
+        }
+
+        private int? age;
+        public int? Age
+        {
+            get => this.age;
+            set {
+                if (value < 1) value = 1;
+                else if (value > 100) value = 100;
+                this.age = value;
```

Comment:
I'd make the getter NOT nullable.  And when you return this.age set it to an actual number (maybe -1?) is it was null

Reply:
`get {
if(this.age == null)
this.Age= 1;
return this.age;
}`

Reply:
so should it be "public int Age" instead of "public int? Age" ? if i do that then I get an error of converting an int? to an int

Reply:
@gregoryfein